data/strategies/42161/CurveGauge.json

```diff
@@ -15,7 +15,7 @@
     },
     "es": {
       "name": "Curve Gauge",
-      "description": "Supplies {{token}} to [Curve Finance](https://arbitrum.curve.fi) and stakes it in gauge to collect any available tokens and earn CRV rewards. Earned tokens are harvested, sold for more {{token}} which is deposited back into the strategy."
+      "description": "Suministra {{token}} a [Curve Finance](https://arbitrum.curve.fi) y hace staking en el indicador para recolectar los tokens disponibles y ganar recompensas en CRV. Los tokens ganados se cosechan, se venden por más {{token}} que se deposita nuevamente en la estrategia."
```

Comment:
I think in the mainnet translations you kept "gauge" without translation, I would keep it untranslated on this one as well for consistency.